### PR TITLE
Minor adjustments

### PR DESCRIPTION
**Pull request by martin-gmx**
Hi Andy,

Some more refactoring done to clean up some more.
Should work without problems...

Cheers
Martin

### CODE REVIEW

**Comment by martin-gmx** [comment]
Added some further refactoring to simplify the code - to later be able to add speedup and other methods for faster derivative calculations.

File: R/get_cv.R [hunk 1]
```diff
@@ -142,7 +146,7 @@ get_rse <- function (fim, poped.db,
   params_rse <- returnArgs[[2]]
   parnam <- get_parnam(poped.db)
   ret <- params_rse[,,drop=T]
-  if(use_percent) ret[params!=0]=ret[params!=0]*100
+  if(use_percent) ret[params!=0]=round(ret[params!=0]*100)
```
Comment (andrewhooker) [inline R/get_cv.R]: I think that rounding results from this function can be dangerous.  I would rather round results when printing.  I'll make some changes here.

File: R/get_cv.R [hunk 2]
```diff
@@ -103,6 +103,10 @@ get_rse <- function (fim, poped.db,
   for(i in names(called_args)[-1]){
     if(length(grep("^poped\\.db\\$",capture.output(default_args[[i]])))==1) {
       #eval(parse(text=paste(capture.output(default_args[[i]]),"<-",called_args[[i]])))
+      if (i %in% c('bpop','d')) {
+        if (eval(parse(text=paste("dim(",i,")[2]>1"))))
+          (eval(parse(text=paste(i, "<-",i,"[,2]"))))
+      }
```
Comment (andrewhooker) [inline R/get_cv.R]: This doesn't do the right thing and should be removed.  I will do this

**Comment by andrewhooker** [comment]
Thanks Martin! 

I have removed the rounding for get_rse ... so if you want to have to output rounded when printing I suggest we add a method with print.rse

**Pull request merged**